cvmfs/catalog_mgr.h

```diff
@@ -191,7 +191,7 @@ class AbstractCatalogManager : public SingleCopy {
   }
   inline CatalogT* GetRootCatalog() const { return catalogs_.front(); }
   /**
-   * Inodes are ambiquitous under some circumstances, to prevent problems
+   * Inodes are ambiguous under some circumstances, to prevent problems
```

Comment:
The tool I used to pick a replacement suggested `ubiquitous` -- I sympathize, but I think this is the right word...